sw/airborne/modules/pca9685/pca9685_i2c.c

```diff
@@ -0,0 +1,467 @@
+/*
+ * Copyright (C) 2011-2013 The Paparazzi Team
```

Comment:
update header and apply `fix_code_style.sh` script

Reply:
FIXED